This patch will enable launching skippy-xd via Mod-Escape. This
is an feature akin to OS X's "Expose" feature which shows all
windows in one view and allows selecting one.

--- config.h	2021-12-11 03:00:23.835325458 -0500
+++ config.h	2021-12-11 03:00:23.675319428 -0500
@@ -67,10 +67,12 @@
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
 static const char *dmenucmd[] = { "/home/dsicilia/dev/suckless/dmenu/latest/dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
 static const char *termcmd[]  = { "/home/dsicilia/dev/suckless/st/latest/st", NULL };
+static const char *skippycmd[]  = { "/home/dsicilia/bin/skippy-xd", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
+	{ MODKEY,                       XK_Escape, spawn,          {.v = skippycmd } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
 	{ MODKEY,                       XK_b,      togglebar,      {0} },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
